src/blocks/frontend/popup/popup.ts

```diff
@@ -136,11 +131,16 @@ class PopupBlock {
 			return false;
 		}
 
-		const buttons = document.querySelectorAll( `a[href='#${ anchor }']` );
+		const buttons = document.querySelectorAll( `a[href='#${ anchor }'], a#${ anchor }` );
```

Comment:
For the both opening/closing ID selectors, it can just be the ID without it being an anchor element as, for example, Button Block adds the ID to the parent element of the `<a>` tag, so it can just be #${ anchor }